src/steps/roofNextSteps.js

```diff
@@ -40,5 +41,13 @@ export default [
       subTitle: "How big is your house?"
     },
     questions: roofContent['dimensions'].questions
+  },
+  {
+    sectionName: 'summary',
```

Comment:
It's important to keep the titles clear, i'm sure you just didn't notice.
Just know that the main title and the sub title are taking place in some places in the app (the steps for example.
```javascript

  {
    sectionName: 'summary',
    sectionComponent: SummarySection,
    labels: {
      mainTitle: 'Price!',
      subTitle: 'This is the price it is going to cost for you'
    }
  }
```